docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -2258,6 +2258,37 @@ On each interface (not uplink)
   exit
 
 
+N2000 Series (N2024P)
+^^^^^^^^^^^^^^^^^^^^^
+
+This configuration was tested on the version: 6.2.1.6
```

Comment:
Lovely. Must refer to his usual tools.

![farmware](https://www.farmwear.ie/products/012B.SS200.BLU_1.jpg)